package.json

```diff
@@ -134,12 +135,11 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
     "flatpickr": "^4.6.6",
-    "liquid-fire": "^0.32.0",
+    "liquid-fire": "^0.34.0",
```

Comment:
Ik ben vrij zeker dat we die gewoon nergens gebruiken ... mag ms ineens weg?

Reply:
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/commit/c65159a1057c51cdad49a39e510a8531ce799183#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R75

Reply:
Ben daar ook redelijk zeker van. Mag weg van mij.

Reply:
https://c.tenor.com/4bYhursxkq8AAAAC/john-cena-are-you-sure-about-that.gif

https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/994238d20b4b1d41326d2a6f1d927544950904b4/app/transitions.js#L5

Reply:
Die liquid fire ben ik tegen gekomen bij ember bump.
in app/transitions.js lijkt code te staan van die package, maar ik heb niet verder gezocht of dat eigenlijk iets doet in onze app.